Showcase-iOSTests/Login/New Group/LoginVCTests.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  LoginVCTests.swift
+//  Showcase-iOSTests
+//
+//  Created by Lehlohonolo Mbele on 2018/05/10.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import XCTest
+import Cuckoo
+@testable import Showcase_iOS
+
+class LoginVCTests: XCTestCase {
```

Comment:
the name of this class. Please can you name it properly. We don't do abbreviations. Please name it LoginViewControllerTests. Naming is important can we please name classes correctly.